### PR TITLE
Revision Cord

### PR DESCRIPTION
**Pull request by ckaldemeyer**

### CODE REVIEW

**Comment by ckaldemeyer** [comment]
Mache morgen weiter..

File: addressing_energy_system_modelling_challenges.tex [hunk 1]
```diff
@@ -338,65 +338,61 @@ \section*{Introduction}
...
-covering long time horizons and large areas \cite{Pfenninger2014}.
-Different types of uncertainty may affect results obtained from energy system
-models \cite{Craig2002, Hunter2013, Mirakyan2015}.
+covering long time horizons and large geographical areas \cite{Pfenninger2014}.
+Different types of uncertainty may also affect results obtained from energy
+system models \cite{Craig2002, Hunter2013, Mirakyan2015}.
 Furthermore, energy is a driver of economic growth \cite{Warr2010} and
 interacts with essential goods like water and food \cite{Chang2016}. Consequently,
-energy research is closely linked to other research disciplines. In
-\emph{modern realities} and the light of resource scarcity and
```
Comment (simnh) [inline addressing_energy_system_modelling_challenges.tex]: Please do not remove modern realities terms throughout the paper... I see the point that a prescise definition needs to be given somewhere at the beginning. But we actually agreed on that it's quite fitting term for the problems/challenges ahead....

**Comment by ckaldemeyer** [comment]
> Please do not remove modern realities terms throughout the paper... I see the point that a prescise definition needs to be given somewhere at the beginning. But we actually agreed that it's quite fitting term for the problems/challenges ahead....

Das war nicht mein Plan. Die Formulierung findet sich aber m. E. manchmal einmal zuviel wieder und dann würde ich eine von zehn rausschmeissen

**Comment by ckaldemeyer** [comment]
Ab dafür!

**Pull request merged**